src/main/docs/guide/security/authenticationStrategies/jwt/reader/bearerToken.adoc

```diff
@@ -1,25 +1,21 @@
-Micronaut supports https://tools.ietf.org/html/rfc6750[RFC 6750 Bearer Token] specification out-of-the-box.
-
-It is enabled by default but requires JWT Authentication to be enabled.
-
-The following sequence illustrates the authentication flow:
+Micronaut supports the https://tools.ietf.org/html/rfc6750[RFC 6750 Bearer Token] specification for transmitting JWT tokens. The following sequence illustrates the RFC 6750 authentication flow:
 
 image::jwt-bearer-token.svg[]
 
-The following configuration properties are available to customize Bearer Token read:
+The following configuration properties are available to customize how the Bearer Token will be embedded in the request:
```

Comment:
Actually, the configurations described are about how the token is read. No how the token is embedded in the request.

Reply:
Thanks, will revert that change.